templates/components/products/add-to-cart.html

```diff
@@ -1,5 +1,5 @@
 <div id="add-to-cart-wrapper" class="add-to-cart-wrapper" {{#unless product.can_purchase}}style="display: none"{{/unless}}>
-    {{#if product.show_quantity_input}}
+    {{#if theme_settings.show_product_quantity_box}}
```

Comment:
@bc-alexsaiannyi Was this change supposed to be reverted here? This change was made in Cornerstone 6.8